### ISSUE TITLE
Integration with Ava

### ISSUE THREAD

**Issue by Cmdv** [testem/testem#765]
Trying to integrate [Ava](https://github.com/sindresorhus/ava) with Testem. Ava isn't browser ready but I wanted to just use Testem in the terminal from Ava's tap output.

I can get both working together but I wondered if I could get more information out than just:

<img width="500" alt="screen shot 2016-01-28 at 22 47 58" src="https://cloud.githubusercontent.com/assets/3036557/12661474/af27e23c-c611-11e5-8986-85dacaec4077.png">

Obviously I could just use Nodemon + Ava + Faucet but I like the way Testem works and if Ava becomes browser ready then they will be a great combination.

My setup is:
```JSON
// .testem.json
{
  "framework": "custom",
  "launchers": {
    "tap": {
      "command": "ava test/*.js --tap",
      "protocol": "tap"
    }
  },
  "launch_in_dev": [ "tap" ]
}
```
```JSON
//package.json
"scripts": {
    "test": "testem",
  }
```
```Js
// test/index.js
import test from 'ava';
test('foo', t => {
  t.pass();
});

test('foo', t => {
  t.pass();
});
```

I should add if the way Testem reports in the terminal can't be changed please close this :)

**Comment by johanneswuerbach** [comment]
Yes, test details should be reported for tap printing processes. Looks like this is broken.

**Comment by Cmdv** [comment]
Ah ok, wasn't entirely sure if I'd set things up incorrectly but if anyone requires more information then let me know

**Comment by elfet** [comment]
Well be cool to have it.

**Comment by markmarijnissen** [comment]
Indeed, this also seems to be broken with Node Mocha tests. I followed instructions in the README.md and while the test run, I see no report.

**Comment by johanneswuerbach** [comment]
Could you try testem master? It should be fixed now and I'm planning a release soon

**Comment by johanneswuerbach** [comment]
The issue should be resolved with testem `v1.11.0`
